images/manageiq-base-worker/Dockerfile

```diff
@@ -9,6 +9,17 @@ LABEL name="manageiq-base-worker" \
 
 COPY container-assets/entrypoint /usr/local/bin
 COPY container-assets/manageiq_liveness_check /usr/local/bin
+COPY container-assets/VMware-* /tmp/
+
+RUN cd /tmp && \
+    filename=$(ls VMware-*) && \
+    [[ -f $filename ]] && \
+    echo "VDDK library found: $filename, installing..." && \
+    tar -zxvf VMware-* && \
+    cp -rf vmware-vix-disklib-distrib/ /usr/lib/vmware-vix-disklib/ && \
+    for SO in $(ls /usr/lib/vmware-vix-disklib/lib64/libvixDiskLib.so*); do ln -s $SO /usr/lib/$(basename $SO); done && \
+    ldconfig && \
+    rm -rf /tmp/VMware-* /tmp/vmware-*
```

Comment:
@bdunne could this live in a helper script that also lives in `container-assets/` like `container-assets/install-vmdk` so that we don't have to one-line this relatively complex bash statement?

Reply:
should it not be `container-assets/install-vddk` or `container-assets/install-vmware-vddk`?

Reply:
:+1: yes `s/vmdk/vddk/` was a typo